app/src/main/java/com/example/cocktails/ui/main/SectionsPagerAdapter.kt

```diff
@@ -16,18 +16,16 @@ private val TAB_TITLES = arrayOf(
  * A [FragmentPagerAdapter] that returns a fragment corresponding to
  * one of the sections/tabs/pages.
  */
-class SectionsPagerAdapter(private val context: Context,
-                           fm: FragmentManager,
-                           val cocktail: Cocktail) :
+class SectionsPagerAdapter(private val context: Context, fm: FragmentManager, private val cocktail: Cocktail) :
     FragmentPagerAdapter(fm, FragmentPagerAdapter.BEHAVIOR_RESUME_ONLY_CURRENT_FRAGMENT) {
 
     override fun getItem(position: Int): Fragment {
         // getItem is called to instantiate the fragment for the given page.
         if (position == 0) { // Recipe tab
-            // return a recipe fragment
+            return RecipeFragment.newInstance(cocktail) as Fragment
         }
         else if (position == 1) {  // AR tab
-            return ARFragment(cocktail)
+//            return ARFragment(cocktail)
```

Comment:
@shahafhermann  I changed it by accident, notice when you push your code :)

Reply:
Thanks for the heads up